prem/DivAndSpan.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>
+        DivAndSpan
+    </title>
+    <style>
+      
+        body {
+            width: 98.5vw;
+            height: 96.5vh;
+            border: 2px solid black;
+        }
+        .main-section {
+            position: absolute;
+            width: 50%;
+            height: 50%;
+            margin: auto;
+            top: 0;

```

Comment:
why top:0, bottom:0, right: 0, left:0? can you please explain?

Reply:
To make the position of the element to the origin position, so that element can be made center by using margin:auto